### DISCUSSION TITLE
My achievements are gone from my profile

### DISCUSSION THREAD

**Discussion by CesarCanoff** [community/community#50552] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Bug

### Body

**Greetings everyone**, I would like to ask what I need to do to get my achievements back on my profile, a few days ago they were there, but yesterday they disappeared.

If someone knows how to solve it, I would be grateful!
Thank you in advance!
Good Greetings

**Comment by radar-blip** [comment]
Have you checked this in your settings?

![achi](https://user-images.githubusercontent.com/39196767/226509587-cda4475c-108e-4e73-a191-fe27254978fe.png)


and this too:

![achi2](https://user-images.githubusercontent.com/39196767/226509986-ef56bfda-545c-4626-9252-ad585967db9a.png)

**Reply by CesarCanoff** [reply]
**Yeah**

![image](https://user-images.githubusercontent.com/82189342/226511770-648623e9-82c8-479b-b989-388d3b1dcb84.png)

**Comment by CodeMaster7000** [comment]
Hi @cesarcanoff It is weird that there is some sort of issue here.

I had a similar issue here (https://github.com/Schweinepriester/github-profile-achievements/issues/657), and I would recommend opening an issue to alert Schweinepriester. He replied in a day and fixed this problem.

**Reply by CesarCanoff** [reply]
Hi @CodeMaster7000,
I have opened the problem as you said, and now I am waiting for answers, I would like to thank you for your time.

If my problem is solved, be sure to mark your comment as a reply!

**Reply by CodeMaster7000** [reply]
Sure!

**Comment by SmashedFrenzy16** [comment]
Hi @cesarcanoff 

This is an issue that is very common on GitHub and has not been solved yet.

First make sure achievements are enabled on your profile in settings. If that does not work then contact [GitHub Support](https://support.github.com/).

Also bear in mind GitHub might be working on it as achievements are still in beta mode.

Hope this helps and have a great day!

**Reply by SmashedFrenzy16** [reply]
Also, if you find this answer useful, then please mark it as an answer!

**Reply by CesarCanoff** [reply]
Hello @SmashedFrenzy16,

I've opened an issue [here](https://github.com/Schweinepriester/github-profile-achievements/issues/691), if my issue is not resolved, I will contact GitHub Support.

Thank you for your time!

**Reply by SmashedFrenzy16** [reply]
Ok, and I am always happy to help!

**Comment by Bigizic** [comment, marked as answer]
Go to https://github.com/{username}?tab=achievements

replace {username} with your username

There's an eye icon at the top left, turn it on

**Reply by BrunoFelixDeSousa** [reply]
It helped a lot, thank you!

**Reply by daviSR99** [reply]
Helped a LOT. I was completely lost on what happened. Thank you!
